### DISCUSSION TITLE
How to change font type ?

### DISCUSSION THREAD

**Discussion by ghost** [TriliumNext/Trilium#2248] [Q&A] [answered]
How can I change the font type ?
Also,  can I use more than font in the same document!?

**Comment by zadam** [comment, marked as answer]
In the options it's possible to override theme fonts:

![image](https://user-images.githubusercontent.com/617641/138043625-b1bdaac9-6233-42f7-9234-802c512567e7.png)

In the same document it's not possible to use different fonts.

**Reply by ghost** [reply]
thanks
